src/campsite-info/access-point-info.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent } from 'react';
+import { CampHeader } from './components/camp-header';
+import { CampImages } from './components/camp-images';
+import { Box } from '@mui/material';
+import { graphicsStyles, mapStyles, imageStyles } from './campsite-info.styles'
+
+export const CampsiteInfo: FunctionComponent = () => {
```

Comment:
Two points on this:
1. We should avoid having two components with the same name if possible
2. If this code is literally the same as the camp info component then we're doing it wrong somehow. We should have a single component that defines the structure that we can feed different data into.